pyexchange/dydx.py

```diff
@@ -140,13 +166,20 @@ def place_order(self, pair: str, is_sell: bool, price: float, amount: float) ->
         ## Retrieve market information for given pair
         market_info = self.get_pair(pair)
         tick_size = abs(Decimal(market_info['minimumTickSize']).as_tuple().exponent)
-        # As market_id is only used for amount, use baseCurrency instead of quoteCurrency
+        # As market_id is used for amount, use baseCurrency instead of quoteCurrency
         market_id = market_info['baseCurrency']['soloMarketId']
+        quote_market_id = market_info['quoteCurrency']['soloMarketId']
+
+        # if USDC adjust price for decimals difference
+        if quote_market_id == consts.MARKET_USDC:
+            price = Decimal(f"{price}e-12")
```

Comment:
`market_info['baseCurrency']['decimals']` will provide the decimal precision (see https://docs.dydx.exchange/#/api?id=get-v2marketsmarket).  Since you're already making the call to get tick size, can't you use that data here and avoid a conditional on token name?

Reply:
Yes, that would be cleaner. I was re-purposing old code, but will adjust accordingly.

Reply:
I think a conditional of some sort would still be necessary - `market_info['baseCurrency']['decimals']` returns the decimal precision, but the price needs to be expressed in terms of 18 decimal wei, so it would have be multiplied by `10 ** -12`

Reply:
Yes that would be cleaner. I've adjusted it accordingly